mne/filter.py

```diff
@@ -659,21 +670,28 @@ def construct_iir_filter(iir_params, f_pass=None, f_stop=None, sfreq=None,
         Wp = f_pass / (float(sfreq) / 2)
         # IT will de designed
         ftype_nice = _ftype_dict.get(ftype, ftype)
+        if phase in ('zero-double', 'zero'):
+            ptype = 'zero-phase (two-pass forward and reverse) non-causal'
+        else:
+            ptype = 'non-linear phase (one-pass forward) causal'
         logger.info('')
         logger.info('IIR filter parameters')
         logger.info('---------------------')
-        logger.info('%s %s zero-phase (two-pass forward and reverse) '
-                    'non-causal filter:' % (ftype_nice, btype))
-        # SciPy designs for -3dB but we do forward-backward, so this is -6dB
+        logger.info('%s %s %s filter:' % (ftype_nice, btype, ptype))
```

Comment:
Nowadays we prefer f-strings
```suggestion
        logger.info(f'{ftype_nice} {btype} {ptype} filter:')
```